test/sys/test_sockopt.rs

```diff
@@ -298,8 +298,15 @@ fn test_tcp_function_blk() {
     let name = unsafe { CStr::from_ptr(tfs.function_set_name.as_ptr()) };
     assert!(!name.to_bytes().is_empty());
 
+    let aliastfs = getsockopt(&fd, sockopt::TcpFunctionAlias).unwrap();
+    let aliasname =
+        unsafe { CStr::from_ptr(aliastfs.function_set_name.as_ptr()) };
+    // freebsd default tcp stack has no alias.
+    assert!(aliasname.to_bytes().is_empty());
+
     // We can't know at compile time what options are available.  So just test the setter by a
     // no-op set.
+    // TODO: test if we can load for example BBR tcp stack kernel module.
```

Comment:
We just need to ensure the wrapper works, so it is not necessary to do further tests, but having them harms nothing:)

Reply:
it is more in case future related flag additions, some of which need this alternative stack to work.